src/include/lean/lean.h

```diff
@@ -1847,6 +1847,137 @@ static inline uint8_t lean_usize_dec_le(size_t a1, size_t a2) { return a1 <= a2;
 static inline uint32_t lean_usize_to_uint32(size_t a) { return ((uint32_t)a); }
 static inline uint64_t lean_usize_to_uint64(size_t a) { return ((uint64_t)a); }
 
+/*
+ * Note that we compile all files with -frwapv so in the following section all potential UB that
+ * may arise from signed overflow is forced to match 2's complement behavior.
+ *
+ * We furthermore rely on the implementation defined behavior of gcc/clang to apply reduction mod
```

Comment:
There is an issue open at LLVM to document its implementation defined behavior https://github.com/llvm/llvm-project/issues/11644 but it's just not being handled since 10+ years :(

Reply:
Okay, maybe just link to that issue in addition to the GCC link you already provided.